src/Common/src/System/Security/Cryptography/DerEncoder.cs

```diff
@@ -532,18 +547,61 @@ internal static byte[][] SegmentedEncodeUtf8String(char[] chars, int offset, int
         /// <returns>The encoded segments { tag, length, value }</returns>
         internal static byte[][] ConstructSegmentedSequence(params byte[][][] items)
         {
+            return ConstructSequenceWithTag(ConstructedSequenceTag, items);
+        }
+
+        private static byte[][] ConstructSegmentedConstructedImplicit(int contextNumber, params byte[][][] items)
+        {
+            Debug.Assert(contextNumber >= 0 && contextNumber < 32);
+            return ConstructSequenceWithTag((byte)(ConstructedImplicitTag | contextNumber), items);
+        }
+
+        internal static byte[][] ConstructSegmentedImplicitSequence(int contextNumber, params byte[][][] items)
+        {
+            return ConstructSegmentedConstructedImplicit(contextNumber, items);
+        }
+
+        internal static byte[][] ConstructSegmentedImplicitSet(int contextNumber, params byte[][][] items)
+        {
+            return ConstructSegmentedConstructedImplicit(contextNumber, items);
+        }
+
+        internal static byte[][] ConstructSegmentedExplicitSequence(int contextNumber, params byte[][][] items)
+        {
+            byte[][] innerSequence = ConstructSegmentedSequence(items);
+            return ConstructSegmentedImplicitSequence(contextNumber, innerSequence);
+        }
+
+        internal static byte[][] ConstructSegmentedExplicitSequenceFromPayload(int contextNumber, params byte[][] items)
+        {
+            byte[][] innerSequence = ConstructSegmentedSequenceFromPayload(items);
+            return ConstructSegmentedImplicitSequence(contextNumber, innerSequence);
+        }
+
+        private static byte[][] ConstructSegmentedFromPayloadWithTag(byte tag, params byte[][] items)
```

Comment:
Since this is just a pass-into from other params methods it doesn't need the params keyword.